### PR TITLE
Added 'mul' function

### PR DESCRIPTION
**Pull request by nav33nb**
> Added **mul** function
> Some changes to improve readability

### CODE REVIEW

File: calc.cpp [hunk 1]
```diff
@@ -2,22 +2,22 @@
 
 using namespace std;
 
+//FUNCTION TO MULTIPLY
 int mul(int num1,int num2){
-  return (num1*num2);
+  int mul = num1*num2;
```
Comment (Gs-001) [inline calc.cpp]: Instead of using a variable to store the result, you can just return the value from the function

Comment (Gs-001) [reply calc.cpp]:
```suggestion
  return (num1*num2);
```
How about this ?

> thou shalt not use thy unnecessary variables

Comment (nav33nb) [reply calc.cpp]: Are brackets Really necessary?

Comment (Gs-001) [reply calc.cpp]: The return keyword does not actually need brackets but we are following this syntax as we also followed it in other functions

Comment (nav33nb) [reply calc.cpp]: Thanks, will do.

**Pull request merged**